src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -315,6 +315,17 @@ public PagedIterable<GHUser> listPublicMembers() throws IOException {
         return listMembers("public_members");
     }
 
+    /**
+     * All the outside collaborators of this organization.
+     *
+     * @return the paged iterable
+     * @throws IOException
+     *             the io exception
+     */
+    public PagedIterable<GHUser> listOutsideCollaborators() throws IOException {
+        return listMembers("outside_collaborators");
```

Comment:
Please also cover this method with a test.  The default filter is `all`. 
 Adding one more call and some verifications to the test you wrote shouldn't be too much work.  And the list returned from the call should be identical to the call already made in your test.

Reply:
DONE